### ISSUE TITLE
Bug/Feature Req: Italics Support?

### ISSUE THREAD

**Issue by typkrft** [ranger/ranger#2503]
#### Runtime Environment

- Operating system and version: 12.0 (21A344)
- Terminal emulator and version: iTerm2 Build 3.4.13
- Python version: 3.10.0 (default, Oct 22 2021, 13:24:07) [Clang 13.0.0 (clang-1300.0.29.3)]
- Ranger version/commit: ranger-master 2.7.4-4237-g354718c40
- Locale: en_US.UTF-8

#### Current Behavior
Using the Configuration below doesn't display italicized text. It's no big deal, but it does work outside of ranger and I didn't see an issue so I thought I'd mention it. Thanks for the great piece of software. 

``` SHELL 
        ## Text
        text/* | */xml)
            ## Syntax highlight
            if [[ "$( stat --printf='%s' -- "${FILE_PATH}" )" -gt "${HIGHLIGHT_SIZE_MAX}" ]]; then
                exit 2
            fi
            if [[ "$( tput colors )" -ge 256 ]]; then
                local pygmentize_format='terminal256'
                local highlight_format='xterm256'
            else
                local pygmentize_format='terminal'
                local highlight_format='ansi'
            fi
            env HIGHLIGHT_OPTIONS="${HIGHLIGHT_OPTIONS}" highlight \
                --out-format="${highlight_format}" \
                --force -- "${FILE_PATH}" && exit 5
            env COLORTERM=xterm-256color bat --theme=dracula_italicized --color=always --wrap=auto --italic-text=always  \
                -- "${FILE_PATH}" && exit 5
            pygmentize -f "${pygmentize_format}" -O "style=${PYGMENTIZE_STYLE}"\
                -- "${FILE_PATH}" && exit 5
            exit 2;;
```
#### Context

On the surface it's almost imperceptible, but deep down aesthetically the pain is almost too great to bear. \s

**Comment by toonn** [comment]
I wonder whether this is because Python curses doesn't support the full ncurses feature set? More than 256 colors don't work either, for example.

**Comment by typkrft** [comment]
I kind of suspected it was a python dependency. https://docs.python.org/3/library/curses.html Shows some documentation mentioning italics starting on 3.7, but I don't know how relevant that particular information is here.

**Comment by toonn** [comment]
Yeah, looks like 3.10 has finally added support for more than 256 colors too. We should really work on these but there's a huge backlog of fixes we have to work on releasing first. Hopefully the beginning of next year will bring some time to work on this. (Help welcome!)
